DailySelfie/app/src/main/res/menu/options_menu.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+<!-- "Take Picture", should appear as action button if possible -->
+<item
+    android:id="@+id/action_take_picture"
+    android:icon="@drawable/ic_add_a_photo_black_24dp"
+    android:title="@string/action_take_picture"
```

Comment:
We name strings with their context in the prefix. for example, in this case, the would be:
@string/activity_main_action_take_picture